test/mock_batch_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:math';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  final usersPath = "users";
+  final firestore = MockFirestoreInstance();
```

Comment:
You are using the old class name, so Analyzer throws this error:
> The function 'MockFirestoreInstance' isn't defined • test/mock_batch_test.dart:7:21 • undefined_function

Also you should initialize a new instance inside every `test()`, because you don't want the instance of a previous test to affect the next test. Especially since tests can be run in random order. See https://pub.dev/packages/test#shuffling-tests.

Please remove this line, and add `final firestore = FakeFirebaseFirestore();` at lines 27 and 47.